acceptance-tests/simple-permissioning-smart-contract/test/test-account-permissioning.js

```diff
@@ -0,0 +1,47 @@
+const TestPermissioning = artifacts.require('SimpleAccountPermissioning.sol');
+var proxy;
+
+var address1 = "0x627306090abaB3A6e1400e9345bC60c78a8BEf57";
+var address2 = "0xfe3b557e8fb62b89f4916b721be55ceb828dbd73";
+
+var value = 99;
+var gasPrice = 88;
+var gasLimit = 77;
+var payload = "0x1234";
+
+contract('Permissioning: Accounts', () => {
+  describe('Function: permissioning', () => {
```

Comment:
There is no need to use `contract()` and `describe()`

https://truffleframework.com/docs/truffle/testing/writing-tests-in-javascript

Reply:
done